lib/src/mock_document_snapshot.dart

```diff
@@ -39,4 +44,17 @@ class MockDocumentSnapshot extends Mock implements DocumentSnapshot {
 
   @override
   SnapshotMetadata get metadata => _metadata;
+
+  bool _isComposedKey(String key) {
+    return key.contains('.');
+  }
+
+  dynamic getComposedKeyValue(String key) {
```

Comment:
I tried searching the Firestore doc and wasn't able to find the official term for it. So I don't know what the proper term is. But for better or for worse, we called those "composite keys" in https://github.com/atn832/cloud_firestore_mocks/blob/d8be0b9c9d522a0d78ebe2303bc5756c09e75951/lib/src/mock_document_reference.dart#L120 and in other places. Would you mind renaming this to `getCompositeKeyValue` for consistency?